src/components/issue-blurb.jsx

```diff
@@ -10,144 +9,153 @@ import GithubFlavoredMarkdown from './gfm';
-            disableLinks={false}
-            repoOwner={repoOwner}
-            repoName={repoName}
-            text={issue.body}/>
-        </BS.Popover>
-      );
+    const bodyPopover = (
+      <BS.Popover id={`popover-${repoOwner}-${repoName}-${number}`} className='issue-body' title={popoverTitle}>
+        <GithubFlavoredMarkdown
+          disableLinks={false}
+          repoOwner={repoOwner}
+          repoName={repoName}
+          text={issue.body}/>
+      </BS.Popover>
+    );
 
-      let icon = null;
-      if (isPullRequest) {
-        let state;
-        if (card.isPullRequestMerged()) {
-          state = 'merged';
-        } else {
-          state = issue.state;
-        }
-        icon = (
-          <GitPullRequestIcon title='Click for Pull Request Details' className='blurb-icon' onClick={this.onClickIcon} data-state={state}/>
-        );
+    let icon = null;
+    if (isPullRequest) {
+      let state;
+      if (card.isPullRequestMerged()) {
+        state = 'merged';
       } else {
-        let iconType;
-        if (issue.state === 'open' && !issue.closedBy) {
-          iconType = IssueOpenedIcon;
-        } else if (issue.state === 'open' && issue.closedBy) {
-          iconType = IssueReopenedIcon;
-        } else if (issue.state === 'closed') {
-          iconType = IssueClosedIcon;
-        } else {
-          iconType = QuestionIcon;
-        }
-        icon = (
-          <iconType title='Click for Issue Details' className='blurb-icon' onClick={this.onClickIcon} data-state={issue.state}/>
-        );
+        state = issue.state;
       }
-
-      const classes = {
-        'issue-blurb': true,
-        'is-pull-request': isPullRequest,
-        'is-merged': card.isPullRequestMerged()
-      };
-
-      let kanbanColumnIcon;
-      const kanbanColumn = issue.labels.filter((label) => {
-        return getFilters().getState().columnRegExp.test(label.name);
-      })[0];
-      if (kanbanColumn) {
-        const {color, name} = kanbanColumn;
-        kanbanColumnIcon = (
-          <ColoredIcon color={color} name={name}>
-            <ListUnorderedIcon/>
-          </ColoredIcon>
-        );
-      }
-
-      return (
-        <span className={classnames(classes)}>
-          {kanbanColumnIcon}
-          <a className='blurb-number-link'
-            target='_blank'
-            href={issue.htmlUrl}
-            >
-            <span className='blurb-number'>{issue.number}</span>
-          </a>
-          <BS.OverlayTrigger
-            rootClose
-            trigger={['click', 'focus']}
-            placement='bottom'
-            overlay={bodyPopover}>
-            <span className='-just-for-overlay-trigger'>
-              {icon}
-              {privateEl}
-              <span className='blurb-secondary-repo'>{multipleRepoName}</span>
-              {blurbContext}
-            </span>
-          </BS.OverlayTrigger>
-        </span>
+      icon = (
+        <GitPullRequestIcon
```

Comment:
this line has had style modifications when there was no syntax modifications.  pls restore to reduce size of `git diff -w`

Reply:
`this.onClickIcon` has been changed to `onClickIcon`. Should that be regarded as syntax modification?